### DISCUSSION TITLE
nucleo-f031k6 question

### DISCUSSION THREAD

**Discussion by lv333ming** [stm32duino/Arduino_Core_STM32#1802] [Q&A] [answered]
test program：
void setup() {
  
  pinMode(LED_BUILTIN, OUTPUT);
}

void loop() {
  digitalWrite(LED_BUILTIN, HIGH);   
  delay(1000);                      
  digitalWrite(LED_BUILTIN, LOW);    
  delay(1000);   

Download to nucleo-f031k6 LED does not flash 。

MeAdd this link in the "Additional Boards Manager URLs" field: https://raw.githubusercontent.com/stm32duino/BoardManagerFiles/master/STM32/package_stm_index.json
nucleo-f031k6 works fine

**Comment by fpistm** [comment, marked as answer]
Hi @lv333ming 
it should be fixed thanks #1787
